manual/add-new-series.md

```diff
@@ -0,0 +1,53 @@
+# How to add new series
+
+This is currently a manual process for MVP.
+
+This is a stepping stone towards creating a background task which lets a developer update the series table in the database.
+
+The final step will be an admin UI that lets admins add new series themselves.
+
+## Get the series reference
+
+Get the series reference from Claire Driver.
+
+Claire will be talking to cataloguing about each series so will know what the series reference should be.
+
+## Create database migration script
+
+The series IDs are not sensitive, so it is OK for them to appear in the [tdr-consignment-api-data] repository.
+
+1. Create a new sql migration file in the `tdr-consignment-api-data` repository to add the new series. See the [adding a migration script] instructions.
+
+2. If the transferring body has not been added to the `Body` table, include a script in the new sql migration file to add it to the table:
+   ```
+    INSERT INTO "Body" ("BodyId", "Name", "Description", "TdrCode") VALUES
+      (uuid_generate_v4(), '[body_id]', '[body_name]', '[tdr_cde]') RETURNING "BodyId" INTO bodyUuid;
+   ```
+   * `[body_id`]: the id for the new transferring body, eg "Ministry of Justice"
+   * `[body_name]`: the name of the new transferring body, eg "Ministry of Justice"
+   * `[tdr_code]`: the code that is used ***internally by the TDR product***. Should be prefixed by "TDR-", eg: "TDR-MOJ"
+      
+   **Note:** The transferring body needs to exist in the `Body` table before the new series can be added to the `Series` table.
+   
+3. Include a script in the new sql migration file adding the new series to the `Series` table:   
+   ```
+    INSERT INTO "Series" ("SeriesId", "BodyId", "Code", "Name", "Description") VALUES
+      (uuid_generate_v4(), [body_uuid], '[series_code]', '[series_name]', '[series_description]');
+   ```
+   * `[body_uuid]`: this is the value from the `BodyId` column in the `Body` table for the transferring body
+   * `[series_code]`: this is the series reference value, eg "LCO 72"
+   * `[series_name]`: this is the series reference value, eg "LCO 72"
+   * `[series_description]`: this is the series reference value, eg "LCO 72"
```

Comment:
Are series name, code and description always the same? Should we get rid of two of them?

Reply:
Yeah, that would be good. I think we've discussed it before, but I can't find any tasks to remove them.

Reply:
Added JIRA task: https://national-archives.atlassian.net/browse/TDR-1240

Reply:
Added JIRA task: https://national-archives.atlassian.net/browse/TDR-1240